FreeRTOS/FreeRTOS-Plus-TCP
# [RELEASE] Update the version number/history/manifest for 2.3.2

Description
-----------
See title

Test Steps
-----------
N/A

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
